frontend/app/src/people/widgetViews/OrganizationDetails.tsx

```diff
@@ -0,0 +1,463 @@
+  padding: 20px 0px;
+  z-index: 100;
+`;
+
+const DetailsWrap = styled.div`
+  width: 100%;
+  min-height: 100%;
+  margin-top: 17px;
+  padding: 0px 20px;
+`;
+
+const UsersCount = styled.h3`
+    font-size: 1.3rem;
+    margin-bottom: 15px;
+`;
+
+const UsersTable = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-top: 25px;
+`;
+
+const TableRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 10px;
+`
+
+const TableHead = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 10px;
+  background: #D3D3D3;
+`;
+
+const ModalTitle = styled.h3`
+    font-size: 1.2rem;
+`;
+
+const Th = styled.div`
+    font-size: 1.1rem;
+    font-weight: bold;
+    min-width: 25%;
+  `;
+
+const ThKey = styled.div`
+    font-size: 1.1rem;
+    font-weight: bold;
+    min-width: 50%;
+  `;
+
+const Td = styled.div`
+    font-size: 0.95rem;
+    min-width: 25%;
+    text-transform: capitalize;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  `;
+
+const TdKey = styled.div`
+    font-size: 0.95rem;
+    min-width: 50%;
+    text-transform: capitalize;
+  `;
+
+const Actions = styled.div`
+    font-size: 0.95rem;
+    min-width: 25%;
+  `;
+
+const CheckUl = styled.ul`
+    list-style: none;
+    padding: 0;
+    margin-top: 20px;
+`;
+
+const CheckLi = styled.li`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0px;
+    margin-bottom: 10px;
+`;
+
+const Check = styled.input`
+    width: 20px;
+    height: 20px;
+    border-radius: 5px;
+    padding: 0px;
+    margin-right: 10px;
+`;
+
+const CheckLabel = styled.label`
+    padding: 0px;
+    margin: 0px;
+`;
+
+const OrganizationDetails = (props: { close: () => void, org: Organization | undefined }) => {
+    const [loading, setIsLoading] = useState<boolean>(false);
+    const isMobile = useIsMobile();
+    const { main, ui } = useStores();
+    const [isOpen, setIsOpen] = useState<boolean>(false);
+    const [isOpenRoles, setIsOpenRoles] = useState<boolean>(false);
+    const [usersCount, setUsersCount] = useState(0);
+    const [disableFormButtons, setDisableFormButtons] = useState(false);
+    const [users, setUsers] = useState<Person[]>([]);
+    const [user, setUser] = useState<Person>();
+    const [bountyRolesData, setBountyRolesData] = useState<any[]>([]);
+    const config = widgetConfigs['organizationusers'];
+
+    const formRef = useRef(null);
+    const isOrganizationAdmin = props.org?.owner_pubkey === ui.meInfo?.owner_pubkey;
+    const schema = [...config.schema];
+
+    const initValues = {
+        owner_pubkey: '',
+    };
+
+    const getOrganizationUsersCount = useCallback(async () => {
+        if (props.org?.uuid) {
+            const count = await main.getOrganizationUsersCount(props.org?.uuid);
+            setUsersCount(count);
+        }
+    }, [main, props.org?.uuid]);
+
+    const getOrganizationUsers = useCallback(async () => {
+        if (props.org?.uuid) {
```

Comment:
instead of doing props.org?.uuid a bunch of times it would make sense to define it higher up as a variable, would make the code look simpler